summer_staj.py

```diff
@@ -25,26 +31,30 @@
 ########
 
 # some global variables to create two csv files on .txt format
-first_outfile = "data/first.txt"
-second_outfile = "data/second.txt"
+first_outfile = "data/first.csv"
+second_outfile = "data/second.csv"
 second_infile = first_outfile
 
+CUSTOM_SETTINGS = { 'DOWNLOAD_DELAY': 0.1,
+                    # 'ITEM_PIPELINES': {'freedom.pipelines.IndexPipeline': 300 }

```

Comment:
this was generating a lot of odd exceptions w/ python 3.5.2. Should be OK with higher versions but I commented it out.